bigtable/tests/instance_admin_integration_test.cc

```diff
@@ -73,6 +73,17 @@ TEST_F(InstanceAdminIntegrationTest, ListInstancesTest) {
   }
 }
 
+/// @test Verify that InstanceAdmin::GetInstances works as expected.
+TEST_F(InstanceAdminIntegrationTest, GetInstancesTest) {
+  // The emulator does not support instance operations.
+  if (UsingCloudBigtableEmulator()) {
+    return;
+  }
+  auto instance = instance_admin_->GetInstance("t0");
```

Comment:
This is not going to work when we test against the production environment.  Can you add a `TODO(#418)` comment to make the test functional as part of the `CreateInstance()` implementation?

Reply:
yes right, instance ID may different